.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gitsubmodule
```

Comment:
@daimesava Thank you for doing this! Does this cause it to update all the submodules daily? There are some submodules that might cause problems if we pull new commits from them (Sai in particular has been an issue in the past), so I'm wondering if we could target dss-deploy-scripts specifically? This is also my first time looking at dependabot config, so apologies if I'm misunderstanding how this works.

Reply:
Daily is the check interval. It will only open a PR if it finds a newer commit for each submodule.

To exclude problematic ones (or all :) ), we can use `ignore`:

https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#ignore

or ignore all and allow only one:

https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#allow

What would be your preference?

Reply:
Awesome! I think just `allow`ing dss-deploy-scripts is probably best.

Reply:
done 👍

Reply:
Amazing, thank you!